sklearn/metrics/tests/test_common.py

```diff
@@ -1152,7 +1155,8 @@ def test_multiclass_sample_weight_invariance(name):
     y_score = random_state.random_sample(size=(n_samples, 5))
     metric = ALL_METRICS[name]
     if name in THRESHOLDED_METRICS:
-        check_sample_weight_invariance(name, metric, y_true, y_score)
+        y_score_norm = y_score / y_score.sum(1, keepdims=True)
```

Comment:
Can you explain why this is needed?

Reply:
In one of the previous discussions in the old PRs, it was determined that the rows of `y_score` used in multiclass had to sum to one. This is checked in `roc_auc_score`. In the other thresholded metrics, `y_score` did not need to sum to one.

Reply:
which arguably is a terrible choice for log loss. Is there an issue about that?